### DISCUSSION TITLE
Crisis in Management - an alternative-start Syndicate storyline

### DISCUSSION THREAD

**Discussion by leklachu** [endless-sky/endless-sky#7131] [Show and tell]
Hi, I'm writing an alternate start as the boss of a failed Syndicate company, which becomes a full Syndicate storyline through the war. And... I hope maybe some of you will like it! It's supposed to be the story of the character development of your captain, while the 'main' events happen in the background: A bit different from main game  - but my writing's not as good as I hoped, so... it's kind of a mix. I hope it's still fun. It's also more wordy than the main game - but I hope not too much.

I'd love it if there's place for it in the main game, once it's finished. So of course part of this is to ask: Would you be interested in that? What sort of things would I need to do to make that work? And any other feedback :-)

Amazinite is already drafting what's planned to be the main Syndicate storyline, over at https://github.com/Amazinite/ES-Syndicate-Campaign , and a large amount of mine depends on the specific alternate start, so I imagine this will always only be a storyline you can reach through alternate start. That said, if Amazinite or whoever's writing the main one want to use ideas from this, you're welcome to!

It's about two-thirds done, and drafted all the way to the end. Given how things are going, I'm hopeful I can finish in a few weeks to a month. I'll release my overview/info files once it's complete, but for now it's just there as the gamefiles so you can find out what happens by playing ;-)

Still to come: an exciting chase through space! And, quite a different ending. (Keeps continuity with the main plot, just you're doing something else.)

I've hosted it as a plugin here: https://github.com/leklachu/endless-sky-syndicate
and here:
[syndicate_v0.2.1.zip](https://github.com/endless-sky/endless-sky/files/9335541/syndicate_v0.2.1.zip)

**Comment by leklachu** [comment]
Version 3 is ready! This takes the once-hapless ex-manager to the end of a great work, and the cusp of another. Only the last section remains: when the Pug make their dramatic entrance (To Be Concluded).

[syndicate_v0.3.zip](https://github.com/endless-sky/endless-sky/files/9484311/syndicate_v0.3.zip)

**Comment by leklachu** [comment]
And finally, version 4. The story is complete. Just a few minor fixes I want to do before considering it 'mostly ready', but it should be playable now from start to finish. (And with compatibility fixes for anyone with a savegame at the end of one of the previous sections.)

Has anyone tried it yet? Anyone like it? Anyone got hordes of bug reports and complaints? ;-)
[syndicate_v0.4.zip](https://github.com/endless-sky/endless-sky/files/9591467/syndicate_v0.4.zip)

**Reply by MasterOfGrey** [reply]
I recommend promoting this on the discord if you haven't already.
https://discord.gg/ZeuASSx

**Reply by leklachu** [reply]
Thanks. Discord keeps requiring my phone number, so I've abandoned it. But if anyone wants to mention this on the discord you're welcome to :-)

**Comment by Hecter94** [comment]
Just FYI, I added this to the community run plugin archive, just so you're aware. 
https://github.com/Hecter94/EndlessSky-PluginArchive/commit/15eb3bbe29387e3aa2d39275ba97bfc6078c9d8c

**Reply by leklachu** [reply]
Thanks!

**Comment by samrocketman** [comment]
I would like to include this with my fork which has several engine changes but also prepackages plugins https://github.com/samrocketman/endless-sky/releases/tag/continuous-with-plugins

**Reply by leklachu** [reply]
Cool! You're most welcome to. I should have a 0.9 version out within the week with the remaining little bits added. (At which point it should be 'complete' except for minor tweaks and bugfixes.)

**Reply by samrocketman** [reply]
Nice :) I actually include plugins as Git submodules so any updates you make to your own Git repository will automatically be pulled into my release when I update it.  Here's all the submodules soon to include yours after some review https://github.com/samrocketman/endless-sky/tree/mining-complete-plugins/plugins

**Reply by samrocketman** [reply]
I have opened a pull request adding missing information necessary for a plugin https://github.com/leklachu/endless-sky-syndicate/pull/4

I've also opened a few issues on your repository for other recommended changes.  The most important is adding a copyright file.  The LICENSE is also non-standard but not critical as long as you have an updated copyright file https://github.com/leklachu/endless-sky-syndicate/issues

**Reply by samrocketman** [reply]
I followed up with a few more pull requests.

**Reply by leklachu** [reply]
Thanks, that's helpful. :-)

**Comment by leklachu** [comment]
Version '9'. This should be 'approximately complete', meaning there's touch-ups and improvements I'd like to do, but as it stands it's a completed, working plugin.

[syndicate_v0.9.zip](https://github.com/endless-sky/endless-sky/files/9725725/syndicate_v0.9.zip)

**Reply by samrocketman** [reply]
I have added it [to my fork and included in the description](https://github.com/samrocketman/endless-sky/releases/tag/continuous-with-plugins).

Here's a screenshot enabled I'm going to try it out.

![2022-10-06_21-47](https://user-images.githubusercontent.com/875669/194449215-32fb22e3-ea95-4bbb-8630-b8b995f985c5.png)

**Reply by leklachu** [reply]
I hope you enjoy it!

**Reply by samrocketman** [reply]
It is amazing.  I loaded it onto my steam deck.

You did a good job with the writing.  I'm talking it up on discord so others try it out

**Reply by leklachu** [reply]
Thanks!

**Comment by samrocketman** [comment]
Does the syndicate storyline end when your debt is paid off or is there more story to be unlocked past that point as an alternate of FW campaign?

**Comment by leklachu** [comment]
That's just the intro :-D

After the war starts, you'll get a minor mission (random probability on going to a spaceport whenever no higher priority mission offers), which will launch you into the main story.

Storyline ends when the war's over and our yellow-shipped 'peacemaking' friends have all gone their way.

**Reply by samrocketman** [reply]
Ok sounds good I started doing FW campaign will that lock me out of syndicate?

**Reply by leklachu** [reply]
No, you can do the FW intro missions. From this start it shouldn't be possible to get far enough in FW to lock yourself out.

**Comment by leklachu** [comment]
I have made a draft PR at https://github.com/endless-sky/endless-sky/pull/8364 for inclusion into the main game. By the way, I wonder what people thought of this story, who have played it right through?

**Comment by movingpictures** [comment]
I must be missing something - the mass debt at start essentially puts you in a position of having to sell the under-equipped Star Queen, then grinding pirates to pay off debt. Nothing interesting or unusual seems to happen... just makes the start way more of a grind than normal.

**Reply by samrocketman** [reply]
Not how I got out.  You can buy equipment to burn credits before the bank gets them.  Build up enough assets in alternate goods that eventually you can sell off the assets to buy a second ship.

It is a pretty interesting way to trade.  You no longer worry about missed payments to crew.  Your goal is to keep your credits as close to zero (buying outfits or other goods) so that you can build up wealth in terms of goods.

If you have a home planet where you want to buy a ship you can unload purchased outfits.  It is a financial race because your assets depreciate but in general I was able to get out of debt even before the tutorial mission finished and then I got a surprise 10m credits.

**Comment by leklachu** [comment]
If you reject the copilot who turns up in the first conversation, then yes. Otherwise, he gives you a series of missions, which eventually pays off just about all your debt. (Unless you go really slow, then the final mission is different and pays about 20%).

You do have to go to the spaceport on the first planet, and the first mission is on a short timer that you must go directly to the destination. I'm afraid it's not the most robust start, but should work as long as you 

- Don't tell the co-pilot you don't want his help any more, in the first couple of missions
- go to the spaceport on each planet you land on (and start on) - the dialogue should make it clear when you need to
- don't fail any of the main mission chain - but they should tell you clearly if you failed any

If this makes it through into the main game I'll have to tighten up the intro story!

**Comment by leklachu** [comment]
Let me know if that doesn't work for you, it's possible there's a bug I missed, or something I thought was obvious when I playtested.
